docs/testcases/View Guided Answers Content_V3.md

```diff
@@ -59,7 +66,51 @@ Guided Answers extension opens in view port
     d. A id icon for the Component filter . Id icon is highlighted in blue as Component filter is applied by default.
 
 
+**Expected Results 4** 
+
+If the Guided Answers extension is already open in the existing view port is given focus. A new instance of the GA extension is not opened the existing instance is re used. 
+
+
 
+### Opening Guided Answers to the side in VSCODE
+1. Open VSCode
+2. Select Code in VSCode Nav bar
+3. Select Settings in drop down menu
+4. Click Settings in menu 
+5. Enter 'Guided Answers' in the setting search input field 
+6. Check/Enable 'SAP>UX>Guided Answer:Open in New Tab 
+7. Close Settings
+8. Open the 'Command Palette'
+9. Enter 'SAP: Open Guided Answers'
+10. Select 'SAP: Open Guided Answers' ( See expected result 1 ) 
+11. Open the 'Command Palette'
+12. Enter 'SAP: Open Guided Answers'
+13. Select 'SAP: Open Guided Answers' ( See expected result 2 ) 
+
+
+### Opening Guided Answers to the side in SBAS
+1. Open Dev Space in SBAS
+2. Click on menu icon in side tool bar
+3. Select File
+4. Select Preferences
+6. Click Settings in menu 
+7. Enter 'Guided Answers' in the setting search input field 
+8. Check/Enable 'SAP>UX>Guided Answer:Open in New Tab 
+9. Close Settings
+10. Open the 'Command Palette'
+11. Enter 'SAP: Open Guided Answers'
+12. Select 'SAP: Open Guided Answers' ( See expected result 1 ) 
```

Comment:
settings need to be set after this step to make it open in a new tab after step 13